### ISSUE TITLE
Legal text overlaps buttons in the login view

### ISSUE THREAD

**Issue by pierredemilly** [thebergamo/react-native-fbsdk-next#365]
Some of my users cannot login as the legal conditions text goes over the the confirmation button in the facebook oauth views. I contacted Facebook support which was not helpful and put the blame on the library and on React Native.

Did anyone experience something similar?

See example below (iPhone 6S)

<img src="https://user-images.githubusercontent.com/1489635/207263252-9734ee45-0a11-4bc4-a1f0-07f1e916cf86.png" width="400" />

**Comment by mikehardy** [comment]
That oauth is in a custom browser / webview client that is controlled from their SDK, I hate to pass the buck since it will just feel like a run-around to say "I know they say it's us, but I think it's them..." and not really lead to progress but...this doesn't seem like something in this SDK wrapper's control.

Path forward to resolve it: reproduce it using their native SDK API only to eliminate this wrapper module as the source of the problem (or confirm it conclusively!) - from their docs it appears to be the work of some 15 minutes (which means 30-60 minutes in real life) https://developers.facebook.com/docs/facebook-login/ios then you'll have a conclusive location on the problem.

**Comment by pierredemilly** [comment]
After a day of testing around, I realised that it was one specific permission that added the extra legal conditions text (related to instagram messaging).

So this is a CSS issue on Facebook's side who did not test their layout with this permission included, on a small screen.

The only I managed to fix it is by creating my own Webview, doing the whole oauth process in it and injecting some javascript to hide the legal text.

Filed the issue with Facebook, maybe they'll fix it 🤞
